README.md

```diff
@@ -263,6 +276,9 @@ AutoImport({
     /* ... */
   ],
 
+  // Generate source map
+  sourceMap: false,
```

Comment:
Actually I just found many of them are no longer used: 8e87473b98591abaafabc53d057b2e83e800ab63

Reply:
i have deleted it. 😎